examples/data_preprocessing/police_killing/data_preprocessing_with_regex.ipynb

```diff
@@ -0,0 +1,948 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Preprocessing using RegEx as rules"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The aim of this project is to find names of people killed by the police in a corpus of news paper articles. The corpus was created by Katherine A. Keith et al. (2017) for a similar task using distant supervision. This dataset contains mentions of people (based on keywords related to “killing” or “police”) who might have been killed by the police. The dataset (the HTML documents scraped in 2016 themselves as well as the already sentence-segmented data) are available on the [project’s website](http://slanglab.cs.umass.edu/PoliceKillingsExtraction/) and on [MinIO]( https://knodle.dm.univie.ac.at/minio/knodle/datasets/police_killing/). \n",
+    "There is a train and a test dataset, both of them containing dictionaries with the following keys:\n",
+    "\n",
+    "-\tdocid: unique identifiers of every mention of a person possible killed by the police\n",
+    "-\tname: the normalized name of the person\n",
+    "-\tdownloadtime: time the document was downloaded\n",
+    "-\tnames_org: the original name of the person mentioned in the document\n",
+    "-\tsentnames: other names in the mention (not of the person possibly killed by the police)\n",
+    "-\tsent_alter: the mention, name of the person possible killed by the policed replaced by “TARGET”, any other names replaced by “POLICE”\n",
+    "-\tplabel: for the training data possibly erroneous labels obtained using weak supervision and gold labels for the test data – in this project, only the labels of the test data will be used\n",
+    "-\tsent_org: the original mention\n",
+    "\n",
+    "\n",
+    "The rules used in this notebook are slightly more complicated RegEx than the wordpairs for the simple rules. The RegEx used as rules should cover all possible ways a sentence can express that a person \"TARGET\" was killed by the police (using different words for killing and police as well as active and passive constructions).\n",
+    "\n",
```

Comment:
it would be good to choose one tutorial as the "main" one (this or `data_preprocessing_with_keywords.ipynb`) and refer to it in the other one. E.g. smth like "We did it the same way as in ... The difference between this tutorial and ... is ..."

Reply:
or mb even better: without any choice, simply make a cross-reference and add this notion in both. So that people don't wonder whether they opened the same tutorial twice by mistake or they are different indeed ;)